### PR TITLE
docs: Change docs after implementation of Ground Powered FADECs

### PR DESCRIPTION
**Pull request by Luz5020**
## Summary

When FADEC GND PWR Buttons are implemented in this [PR](https://github.com/flybywiresim/a32nx/pull/7919) the Docs should be changed regarding the Buttons as they would no longer be INOP but useable on the Experimental Version
### Location

https://docs.flybywiresim.com/pilots-corner/a32nx-briefing/flight-deck/ovhd-aft/maintenance/

Discord username (if different from GitHub):
Luz 5020 #3258

### CODE REVIEW

File: docs/pilots-corner/a32nx-briefing/flight-deck/ovhd-aft/maintenance.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ The maintenance crew uses this pushbutton to reset the control circuit, after th
     - The FADEC alternator does not supply the FADEC.
 
 !!! info ""
-    Currently not available or INOP in the FBW A32NX for Microsoft Flight Simulator.
+    Currently only available in the experimental Version.
```
Comment (Sleinmaster) [inline docs/pilots-corner/a32nx-briefing/flight-deck/ovhd-aft/maintenance.md]:
```suggestion
    Currently only available in the experimental version.
```

**Comment by Maximilian-Reuter** [comment]
I dont see why my PR needs to go into EXP. The changes are quite small and should be no issue to be merged into master.

**Comment by Luz5020** [comment]
Gotcha, I‘ll change the wording. Do we even need the notice if it‘s on dev and exp? I‘ll add it to be safe

**Comment by frankkopp** [comment]
> Gotcha, I‘ll change the wording. Do we even need the notice if it‘s on dev and exp? I‘ll add it to be safe

Normally we do not even change docs if a feature is only on Experimental. Experimental is a QA and test version. It also allows us time to update docs. 

When it is on Dev you don't need the admonition at all and it can be removed.

**Comment by Luz5020** [comment]
> > Gotcha, I‘ll change the wording. Do we even need the notice if it‘s on dev and exp? I‘ll add it to be safe
> 
> Normally we do not even change docs if a feature is only on Experimental. Experimental is a QA and test version. It also allows us time to update docs.
> 
> When it is on Dev you don't need the admonition at all and it can be removed.
 
 Alright I’ll make that change later thanks for the help

**Comment by Luz5020** [comment]
Upstream PR is ready to merge, so this can go Rtr now

**Comment by Valastiri** [comment]
Adding just to see status of a32nx pr easier

- https://github.com/flybywiresim/a32nx/pull/7919

**Comment by Valastiri** [comment]
Unfortunately just realized this will need an admonition since it's not on stable. Preferably right below the section header

A simple would suffice:
```
!!! warning ""
	Not available in the Stable Version
```

**Comment by Luz5020** [comment]
I have explicitly asked in this PR, as it was planned as Admin before, I was told to remove it lmao, change will be made dw

**Comment by Luz5020** [comment]
It‘s good now, feel free to merge it

**Pull request merged**